firecrown/models/ccl_density.py

```diff
@@ -0,0 +1,108 @@
+"""Critical and Mean Density  Mass Module - Implementation of the NumberDensity
+abstract class for a critical and mean mass definition.
+========================================
+Class for mass definitions with Delta=200 times the critical/mean density.
+The implemented functions use PyCCL library as backend.
+"""
+from __future__ import annotations
+import pyccl
+from .number_density import NumberDensity
+
+SUPPORTED_CRIT_DENS_FUNC_NAMES = {
+    "Despali16": pyccl.halos.MassFuncDespali16,
+    "Bocquet16": pyccl.halos.MassFuncBocquet16,
+}
+SUPPORTED_MEAN_DENS_FUNC_NAMES = {
+    "Tinker08": pyccl.halos.MassFuncTinker08,
+    "Tinker10": pyccl.halos.MassFuncTinker10,
+}
+
+
+class CCLDensity(NumberDensity):
```

Comment:
`CCLDensity` sounds very generic. Do we anticipate to have in there other types of "densities" from CCL, that are not from the halo mass functions? If not, I think it would be clearer to explicitely say "halo mass function" or "halo density" so we know what this refers to.

Reply:
I think that `HaloMassFunction` or `ClusterMassFunction` would be a better name.     What do you think @vitenti @marcpaterno  ?